src/main/java/svc/data/citations/filters/CitationFilter.java

```diff
@@ -59,7 +59,7 @@ private boolean courtDateFilter(Citation citation){
 	private boolean lastNameFilter(Citation citation, String lastName){
 		boolean keepCitation = true;
 		if (lastName != null  && !lastName.isEmpty()){
-			if (!lastName.equals(citation.last_name)){
+			if (!lastName.toLowerCase().equals(citation.last_name.toLowerCase())){
```

Comment:
Just wanted to let you know that there is a "equalsIgnoreCase" method that would trim this  up